imap_processing/tests/conftest.py

```diff
@@ -15,6 +19,119 @@ def _set_global_config(monkeypatch, tmp_path):
     )
 
 
+@pytest.fixture(scope="session")
+def monkeypatch_session():
+    from _pytest.monkeypatch import MonkeyPatch
+
+    m = MonkeyPatch()
+    yield m
+    m.undo()
+
+
```

Comment:
This isn't being used currently, can we remove it? It looks like you are using the monkeypatch fixture directly below instead of the "session" based one here.
\